src/renderer/transforms/forge.ts

```diff
@@ -15,11 +15,11 @@ export async function forgeTransform(files: Files): Promise<Files> {
 
       // devDependencies
       parsed.devDependencies = parsed.devDependencies || {};
-      parsed.devDependencies['@electron-forge/cli'] = '6.0.0-beta.22';
-      parsed.devDependencies['@electron-forge/maker-deb'] = '6.0.0-beta.22';
-      parsed.devDependencies['@electron-forge/maker-rpm'] = '6.0.0-beta.22';
-      parsed.devDependencies['@electron-forge/maker-squirrel'] = '6.0.0-beta.22';
-      parsed.devDependencies['@electron-forge/maker-zip'] = '6.0.0-beta.22';
+      parsed.devDependencies['@electron-forge/cli'] = '6.0.0-beta.34';
+      parsed.devDependencies['@electron-forge/maker-deb'] = '6.0.0-beta.34';
+      parsed.devDependencies['@electron-forge/maker-rpm'] = '6.0.0-beta.34';
+      parsed.devDependencies['@electron-forge/maker-squirrel'] = '6.0.0-beta.34';
+      parsed.devDependencies['@electron-forge/maker-zip'] = '6.0.0-beta.34';
```

Comment:
This is the fast fix. Better fixes:

* caret the version
* Grab the Forge version from Fiddle's `package.json` file